dashboard/app/access_test.go

```diff
@@ -42,7 +42,7 @@ func TestAccessConfig(t *testing.T) {
 }
 
 // TestAccess checks that all UIs respect access levels.
-// nolint: funlen
+// nolint: funlen, goconst
```

Comment:
We wanted to disable `goconst` for all `_test.go` files?

Reply:
https://github.com/google/syzkaller/pull/4294 was a true positive case.
I think it is helpful.

Reply:
As discussed offline: ok, let's try.

Reply:
Let's land #4285 and check result.